src/api.js

```diff
@@ -1,9 +1,17 @@
 import express from "express";
+import { getReviewsForMovieWithId } from "./reviewList.js";
 
 const apiRouter = express.Router();
 
 apiRouter.get("/movies/:id", (req, res) => {
 	res.send(`movie with id of ${req.params.id}`);
 });
 
+apiRouter.get("/movies/:id/reviews", async (req, res) => {
+	const page = req.query.page;
```

Comment:
Could also be written `const { page } = req.query`, a syntax which is preferred by many.